CMake/Findtinygltf.cmake

```diff
@@ -0,0 +1,26 @@
+find_path(tinygltf_INCLUDE_DIR tiny_gltf.h
```

Comment:
Can we not go up a level from this path?

Reply:
Why can't the files be installed in a `tinygltf` folder?

Reply:
I guess the main concern would be conflicts for nlohmann (#13952) at this point.  I can't tell just looking at the CMake if we'd have issues.  We can worry about that later.